code/stable/ssp/SRS/SSP_SRS.tex

```diff
@@ -77,7 +77,7 @@ \subsection{Table of Symbols}
 \\
 $f$ & Interslice Normal to Shear Force Ratio Variation Function: magnitude of interslice forces as a function of the x coordinatefor interslice index i; can be constant or a half-sine & --
 \\
-${F_{S}}$ & Factor of Safety: The global stability of a surface in a slope & --
+${F_{S}}$ & Factor of Safety: The global stability metric of a slip surface of a slope, defined as the ratio of resistive shear force to mobilized shear force & --
 \\
```

Comment:
In the future, I think we will make factor of safety a more general concept.  In structural engineering this concept is certainly not limited to slope stability analysis.  However, we have enough changes happening at the moment.  We should postpone this conversation for the future.